types/src/transaction/mod.rs

```diff
@@ -414,10 +414,10 @@ pub struct SignedTransaction {
     /// Public key and signature to authenticate
     authenticator: TransactionAuthenticator,
 
-    /// A cached serialization of the raw transaction bytes.
-    /// Prevents serializing the same transaction multiple times.
+    /// A cached size of the raw transaction bytes.
+    /// Prevents serializing the same transaction multiple times to determine size.
     #[serde(skip)]
-    bytes: OnceCell<Vec<u8>>,
+    size: OnceCell<usize>,
```

Comment:
@zekun000 I didn't see the `bytes` being used anywhere except for the size. And it's significant overhead to hold this copy.

It does seem like there is an opportunity to re-use this value when doing BCS for the messages, but it doesn't seem too straightforward.

Reply:
re-using the value is probably not hard, the tricky part is that limited us to the bcs serialization, if we ever switch to use json it'll break